scripts/test.ps1

```diff
@@ -177,10 +184,10 @@ function Invoke-Test
 
                 Reset-TestEnvironment
 
-                if ($output[-2].Contains("Test Run Successful.")) {
-                    Write-Log ".. . $($output[-3])"
+                if ($output[-3].Contains("Test Run Successful.")) {
```

Comment:
Since we have added trx in our test, after summary it will add trx file info which is increasing the result by one line. Since every time we are deleting the trx file before running test, its equal to clean machine run